src/azul/deployment.py

```diff
@@ -91,18 +91,15 @@ def es_endpoint(self) -> Optional[Netloc]:
 
     @property
     def es_instance_count(self) -> Optional[int]:
-        if config.es_endpoint:
-            return config.es_instance_count
+        return config.es_instance_count
 
     @property
     def lambda_env(self) -> Mapping[str, str]:
         if config.share_es_domain:
-            endpoint = "${data.aws_elasticsearch_domain.azul_es_domain.endpoint}"
-            instance_count = "${data.aws_elasticsearch_domain.azul_es_domain.cluster_config[0].instance_count}"
+            instance_count = "${var.azul_es_domain.cluster_config[0].instance_count}"
```

Comment:
Since the variable is now always set, I suspect that `instance_count` can just _always_ be set to this value.

Reply:
It makes sense since ES instance count gets called directly in the `ES.template` so it would be provisioned by terraform either way. Good catch.